ios/BUILD.gn

```diff
@@ -45,6 +45,7 @@ brave_core_public_headers = [
   "//brave/ios/browser/brave_wallet/brave_wallet_factory_wrappers.h",
   "//brave/ios/browser/skus/skus_sdk_factory_wrappers.h",
   "//brave/ios/browser/api/bookmarks/brave_bookmarks_api.h",
+  "//brave/ios/browser/api/ipfs/ipfs_api.h",
```

Comment:
This can be removed and the add `import("//brave/ios/browser/api/ipfs/headers.gni")` and `headers += ipfs_public_headers` below with the rest